protos/protos/public/modeldb/versioning/VersioningService.proto

```diff
@@ -479,7 +479,7 @@ message GetUrlForBlobVersioned {
 
     message Response {
         string url = 1;
-        bool multipart_upload_ok = 3;  // for backcompat: new Client, old Backend
+        bool multipart_upload_ok = 2;  // for backcompat: new Client, old Backend
```

Comment:
I actually picked `3` intentionally to match the proto [in `CommonService`](https://github.com/VertaAI/modeldb/blob/c98b1fe/protos/protos/public/modeldb/CommonService.proto#L95)!
And I have already regenerated the protos for this, so I think it may be too late to change it to `2` anyway 😦 
```suggestion
        bool multipart_upload_ok = 3;  // for backcompat: new Client, old Backend
```

Reply:
Ok, No issue. I can change it will works because the backend does not depend on it.